pkgs/development/tools/build-managers/conan/default.nix

```diff
@@ -16,39 +16,31 @@ let newPython = python3.override {
         sha256 = "1080pdxrvnkr8i7b7bk0dfx6cwrkkzzfaranl7207q6rdybzqay3";
       };
     });
-    astroid = super.astroid.overridePythonAttrs (oldAttrs: rec {
-      version = "1.6.5";
+    future = super.future.overridePythonAttrs (oldAttrs: rec {
+      version = "0.16.0";
       src = oldAttrs.src.override {
         inherit version;
-        sha256 = "fc9b582dba0366e63540982c3944a9230cbc6f303641c51483fa547dcc22393a";
+        sha256 = "1nzy1k4m9966sikp0qka7lirh8sqrsyainyf8rk97db7nwdfv773";
       };
     });
-    pylint = super.pylint.overridePythonAttrs (oldAttrs: rec {
-      version = "1.8.4";
+    tqdm = super.tqdm.overridePythonAttrs (oldAttrs: rec {
+      version = "4.28.1";
       src = oldAttrs.src.override {
         inherit version;
-        sha256 = "34738a82ab33cbd3bb6cd4cef823dbcabdd2b6b48a4e3a3054a2bbbf0c712be9";
+        sha256 = "1fyybgbmlr8ms32j7h76hz5g9xc6nf0644mwhc40a0s5k14makav";
       };
-
     });
   };
 };
 
 in newPython.pkgs.buildPythonApplication rec {
-  version = "1.6.0";
+  version = "1.9.1";
```

Comment:
Why not the most recent version?

Reply:
Version 1.9.2 is the latest, but since I did not test it I did not add it. I could if preferred.

Reply:
Is 1.11.2 a stable release? https://github.com/conan-io/conan/releases

Reply:
Yup, and I have updated the package to reflect that. Thanks!